Dockerfile

```diff
@@ -1,3 +1,24 @@
+# Build libdispatch for the strip binary
+FROM --platform=linux/amd64 debian:12-slim AS libdispatch-build
```

Comment:
These changes install a bunch of runtime dependencies needed by `strip` and `ld`